features/providers/mtr_accelerated/main_dwelling.feature

```diff
@@ -0,0 +1,126 @@
+Feature: Change applicant email addressMain dwelling place changes for MTR-Accelerated measures
```

Comment:
Should 'Change applicant email address' be removed here?

Reply:
Yes, good spot!
That was not a successful rebase! 🤦‍♂️